lib/tjbot.js

```diff
@@ -360,14 +360,14 @@ TJBot.prototype._createServiceAPI = function(service, credentials) {
             break;
 
         case 'language_translator':
-            assert(credentials.hasOwnProperty('username'), "credentials for the " + service + " service missing 'username'");
-            assert(credentials.hasOwnProperty('password'), "credentials for the " + service + " service missing 'password'");
-
-            var LanguageTranslatorV2 = require('watson-developer-cloud/language-translator/v2');
-            this._languageTranslator = new LanguageTranslatorV2({
```

Comment:
how safe is it to remove the username/password authentication option? will this break existing peoples' tjbots if they are using an older version of the language translator service? (i.e. do they need to delete / re-create the service in bluemix?)  or, should we support both authentication modes -- if username/password are specified, use that, otherwise if iam_apikey is specified, use that.